FirebaseAppCheck/Sources/Public/FirebaseAppCheck/FIRAppCheck.h

```diff
@@ -28,21 +28,22 @@ NS_SWIFT_NAME(AppCheck)
 
 /// Returns a default instance of `AppCheck`.
 /// @return An instance of `AppCheck` for `FirebaseApp.defaultApp()`.
-/// @throw Throws an exception if the default app is not configured yet or required  `FirebaseApp`
+/// @throw Throws an exception if the default app is not configured yet or required `FirebaseApp`
 /// options are missing.
 + (instancetype)appCheck NS_SWIFT_NAME(appCheck());
 
 /// Returns an instance of `AppCheck` for an application.
 /// @param firebaseApp A configured `FirebaseApp` instance if exists.
 /// @returns An instance of `AppCheck` corresponding to the passed application.
-/// @throw May throws an exception if required `FirebaseApp` options are missing.
+/// @throw Throws an exception if required `FirebaseApp` options are missing.
 + (nullable instancetype)appCheckWithApp:(FIRApp *)firebaseApp NS_SWIFT_NAME(appCheck(app:));
 
-/// Set an instance of a Firebase app check provider factory. An instance of
-/// `DeviceCheckProviderFactory` is used by default. An instance of `AppCheckDebugProvider` can be
-/// used to test on a simulator on a local machine or a build server.
-/// NOTE: Make sure to call the method before `FirebaseApp.configure()`. If the method is called
-/// after configuring Firebase, the changes will not be applied.
+/// Sets the App Check provider. An instance of `DeviceCheckProviderFactory` is
+/// used by default, but you can use an instance of `AppCheckDebugProvider` to
```

Comment:
It should be `AppCheckDebugProviderFactory`.  Also, I realized that it reads to me like there are only two options either `DeviceCheckProviderFactory` or `AppCheckDebugProviderFactory`, but in fact developers can create their own factories e.g. to create different App Check providers for different Firebase apps in their application. Do you think we can word it better?